DEV/org.openl.core/src/org/openl/binding/impl/method/MethodSearch.java

```diff
@@ -37,38 +43,54 @@ public final class MethodSearch {
     private MethodSearch() {
     }
 
-    private static final int[] NO_MATCH = new int[0];
+    private static final Match NO_MATCH = new Match(null, null, null, null, null, null, null, null, false, null);

```

Comment:
Does it make sense to create a default constructor to avoid calling parametrised constructor with a lot of nulls in?

Reply:
The default constructor adds a lot of lines of code and doesn't make sense for a private class. If fields are final then you have to assign null values for all fields in the default constructor.